### PR TITLE
Re-add /fileqcs-only endpoint, and fix the metric for getting the FPR latest update time

### PR DESCRIPTION
**Pull request by callunity**
GP-3624

### CODE REVIEW

File: swagger.json [hunk 1]
```diff
@@ -305,9 +305,9 @@
           "description": "File Path",
           "type": "string"
         },
-        "qcstatus": {
-          "description": "FileQC status (PASS, FAIL, PENDING)",
-          "type": "string"
+        "qcpassed": {
+          "description": "Is FileQC PASS? (null = PENDING)",
```
Comment (avarsava) [inline swagger.json:309]: why isn't PENDING also false?

Comment (callunity) [reply swagger.json]: There are three states: PASS (true), FAIL (false), PENDING (no value). Although now that I'm thinking it through, no FileQCs will have PENDING value (only PASS/FAIL) so I've removed this part of the comment.

Comment (callunity) [reply swagger.json]: (PENDING only shows up in the endpoints that request data from both FileQC and FPR, to indicate matching FPR records that have no FileQC value. This endpoint is FileQCs only, and FileQCs will only ever have qcpassed = true/false)

File: components/fileqcs/fileQcsController.js [hunk 2]
```diff
@@ -170,14 +170,9 @@ const deleteManyFileQcs = async (req, res, next) => {
 };
 
 /** success returns the last time that the File Provenance Report was imported into SQLite */
```
[the remark this answers is not in the document]
Comment (callunity) [reply components/fileqcs/fileQcsController.js]: Updated

**Review by mlaszloffy — APPROVED** [review]

**Pull request merged**